### ISSUE TITLE
Yhdistele tuotavat tietueet jo olemassa oleviin.

### ISSUE THREAD

**Issue by johannaraisa** [KohaSuomi/koha-plugin-import-remote-biblios#1]
Kohan välivarastoon tullut tietue jyrää jo olemassa olevan tietueen päälle, jos vastaavuus löytyy, tässä pitäisi jotenkin yhdistää vanha ja tuleva tietue.

**Comment by AnneliO** [comment]
Tuo tietueiden tuonti välivaraston kautta ei taida oletuksena tukea yhdistelyä. Ainakin, jos katsoo, mitä vaihtoehtoja käyttöliittymä tarjoaa:

![kuva](https://github.com/user-attachments/assets/7f2cf2dd-3f5c-46a8-b9c3-7e27a0466022)

**Comment by johannaraisa** [comment]
Ei, se tuossa onkin vähän ongelma koska se vain korvaa. Pitäisi jotenkin yrittää se tehdä ennen kuin tulee välivarastoon.

**Comment by johannaraisa** [comment]
Tämä on ongelmallinen toteuttaa, koska ei voi tietää kumpi on parempi. Suljen tämän tiketin.
